specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -337,6 +337,76 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/offlineRegion": {
+      "post": {
+        "operationId": "DatabaseAccounts_OfflineRegion",
+        "x-ms-examples": {
+          "CosmosDBDatabaseAccountOfflineRegion": { "$ref": "./examples/CosmosDBDatabaseAccountOfflineRegion.json" }
+        },
+        "description": "Offline the specified region for the specified Azure Cosmos DB database account.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/regionParameterForOnlineOffline"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The offline region operation is completed successfully."
+          },
+          "202": {
+            "description": "Accepted. The offline region operation will complete asynchronously."
+          }
+        }
```

Comment:
seems this is a long running operation, if so please add  `"x-ms-long-running-operation": true` attribute for both operations

Reply:
fixed.